src/Data/Repositories/ChannelRepository.cs

```diff
@@ -22,8 +22,12 @@
 using FundsManager.Data.Repositories.Interfaces;
 using FundsManager.Jobs;
 using FundsManager.Helpers;
-using Quartz;
+ using FundsManager.Services;
+ using Grpc.Core;
+ using Lnrpc;
+ using Quartz;
 using Microsoft.EntityFrameworkCore;
+ using Channel = FundsManager.Data.Models.Channel;
```

Comment:
I don't know why Rider adds this space before the "using", it happens to me as well

Reply:
Happening to me too. It is a pain in the ***